claudette/core.py

```diff
@@ -53,7 +55,21 @@
 all_models = list(model_types)
 
 # %% ../00_core.ipynb
-models, models_aws, models_goog = all_models[:4], all_models[3:7], all_models[7:]
+models = all_models[:5]
+
+# %% ../00_core.ipynb
+models_aws = ['claude-3-5-haiku-20241022',
```

Comment:
Split these out as their own lists otherwise it's too easy to introduce an indexing bug upon inserting models when slicing `all_models`.